src/clj/ataru/applications/excel_export.clj

```diff
@@ -259,10 +257,19 @@
       (> (count name) 30)
       (subs 0 30))))
 
+(defn get-haku-name [tarjonta-service haku-oid]
```

Comment:
Looks like these (`get-haku-name` and `get-hakukohde-name`) could even be lifted to tarjonta-service, since they're used in a lot of places.